Microsoft.HealthVault.Fhir/Codes/HealthVault/HealthVaultVitalStatisticsCodes.cs

```diff
@@ -15,14 +16,14 @@ namespace Microsoft.HealthVault.Fhir.Codes.HealthVault
     /// </summary>
     public static class HealthVaultVitalStatisticsCodes
```

Comment:
I am experimenting with it right now. I believe this is going to go away. Once I do BloodPressure I will be more clear on the right pattern.

---
In reply to: [123842410](https://github.com/Microsoft/healthvault-fhir-library/pull/3#discussion_r123842410) [](ancestors = 123842410)